README.MD

```diff
@@ -14,19 +12,235 @@ library gives your ASP.NET web application access to all of Stormpath's features
 - Secure API key authentication for your service.
 - Servlet support for Java web applications.
 
+In addition to these core Stormpath features, this SDK provides:
+
+- Support for .NET 4.5 and later
+- Support for Mono on Linux
+- Easy installation using NuGet
+- Fully asynchronous design using the [TPL](https://msdn.microsoft.com/en-us/library/dd460717(v=vs.110).aspx)
+
 If you have feedback about this library, please get in touch and share your
 thoughts! support@stormpath.com
 
 ## Current status
 
-This library is currently under initial development. If you would like to be notified when we publish our first release candidate, watch this repository or send an email to nate@stormpath.com.
+This library is currently under active development. If you would like to be notified when we publish our first release candidate, watch this repository or send an email to nate@stormpath.com.
```

Comment:
We should funnel them to support@stormpath.com.  This is easier for the team to search, and aggregate information.  This isn't possible from your email.